doc/getting_started/installation.rst

```diff
@@ -25,7 +25,7 @@ with the classic notebook, see :ref:`installation-notebook`.
 
 To install JupyterLab and Altair with pip, run the following commands::
 
-    $ pip install jupyterlab altair
+    $ pip install jupyterlab altair[dev]
```

Comment:
I think we should recommend that users do the simple ``altair`` installation, not the full development installation (otherwise, what's the point of splitting them out?)

Reply:
I ovelooked the jupyterlab before altair...so you are right!